gtrending/fetch.py

```diff
@@ -77,10 +77,10 @@ def fetch_developers(language: str = "", since: str = "daily") -> dict:
 
 
 def languages_list() -> list:
-    """Fetch languages
+    """Fetch programming languages.
```

Comment:
If you want the docstring to be this, then please update that part in the readme as well